features/manage-position/Redeem.tsx

```diff
@@ -103,7 +105,7 @@ const Redeem = () => {
         setSuccess(null);
         setError(null);
         try {
-          const tokensToRedeemWei = toWei(tokens);
+          const tokensToRedeemWei = toWeiSafe(tokens);
```

Comment:
I know it defaults but why are you not feeding in decimals here?

Reply:
Ah great catch thanks